src/managers/TileManager.ts

```diff
@@ -38,16 +38,72 @@ class TileManager {
                 ),
                 session.seed,
             );
-            this.list.forEach((tile: Tile) => {
-                const state: TileState = TileState.Unused; // TODO
+
+            for (let i = 0; i < this.list.length; i++) {
+                const tile: Tile = this.list[i];
                 const dto: Nullable<TileDTO> =
                     dtos.find((dto: TileDTO) => dto.id === tile.id) || null;
+
+                let state: TileState = this.identifyTiles(
+                    dto,
+                    session.turnPlayer!,
+                    i,
+                );
                 tile.apply(state, dto);
-            });
+            }
+            this.getTilesInHand(); //TODO where to save/use
             this.onSync.emit();
         });
     }
 
+    private identifyTiles(
+        dto: Nullable<TileDTO>,
+        turnPlayer: int,
+        i: int,
+    ): TileState {
+        if (dto !== null) {
+            return TileState.Placed;
+        }
+        if (
+            i <
+            turnPlayer +
+                this.getStartOfGameTilesAmount(GeneralManager.getPlayers.length)
+        ) {
+            return TileState.Drawn;
+        }
+        return TileState.Unused;
+    }
+
+    private getTilesInHand(): TileDTO[] {
+        const playerAmount: int = GeneralManager.getPlayers.length;
+        const orderIndex: int = PlayerManager.getMe()!.orderIndex!;
+        const allTiles: TileDTO[] = GeneralManager.getTiles()!;
+        const tilesInHand: TileDTO[] = [];
+
+        for (let i = 0; i < allTiles.length; i++) {
+            if (
+                allTiles![i].state === TileState.Drawn &&
+                i % playerAmount === orderIndex
```

Comment:
Distribute cards by giving each player one until everyone has enough, and NOT give the correct amount right away?

Reply:
This pr is not for distributing tiles. Its for identifying at any point in the game what the states of the tiles are. This is important so players can rejoin a lobby